global-script.lua

```diff
@@ -2013,170 +2013,176 @@ function resetPiece(object, rotation, depth)
     return object
 end
 ------
+scaleFactors = {
+    -- Note that we scale the boards up more than the position, so the gaps
+    -- don't increase in size.
+    [true]={name = "Large", position = 1.09, size = 1.1},
```

Comment:
maybe there's something i'm not seeing here but with is size 10% bigger but position is only 9% bigger?

Reply:
Yes. If we scale them up the same, then the gaps between boards will also grow by about the same amount. When I first wrote this, some experimentation seemed to indicated that these two ratios led to a reasonably small gap.